packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -20,20 +20,24 @@ import {
   print,
   visit,
   OperationDefinitionNode,
-  GraphQLType,
   ValidationRule,
   FragmentDefinitionNode,
   DocumentNode,
   GraphQLError,
   IntrospectionQuery,
   getIntrospectionQuery,
+  GraphQLNamedType,
 } from 'graphql';
 import copyToClipboard from 'copy-to-clipboard';
 import {
   getFragmentDependenciesForAST,
   getOperationFacts,
   VariableToType,
 } from 'graphql-language-service';
+import { SchemaReference } from 'codemirror-graphql/src/utils/SchemaReference';
```

Comment:
hmm, perhaps we should make this a top-level public export in `codemirror-graphql` to set a good example.  @imolorhe  does this make sense to you?

Reply:
As far as I can tell, `codemirror-graphql` currently doesn't have any top-level imports at all. Happy to follow up on this in another PR if it makes sense to you folks 🤗